db/mocks.go

```diff
@@ -46,16 +46,13 @@ type DatabaseMap interface {
 	OneSelector
 	Inserter
 	SelectExecer
-	Begin() (*gorp.Transaction, error)
+	Begin() (Transaction, error)
 }
 
 // Transaction offers the combination of OneSelector, Inserter, SelectExecer
 // interface as well as Delete, Get, and Update.
 type Transaction interface {
-	OneSelector
-	Inserter
-	SelectExecer
-	Delete(...interface{}) (int64, error)
-	Get(interface{}, ...interface{}) (interface{}, error)
-	Update(...interface{}) (int64, error)
+	gorp.SqlExecutor
```

Comment:
`SqlExecutor` is a broader type than our current `Transaction`. For instance, it includes `SelectInt`, `SelectNullInt`, etc. Is there a reason to switch to gorp's interface type? FWIW I think it's good Go practice to define narrower interfaces over other packages' types, to make mocking easier.